cinema/serializers.py

```diff
@@ -51,14 +60,15 @@ class Meta:
         fields = ("id", "show_time", "movie", "cinema_hall")
 
 
-class MovieSessionListSerializer(MovieSessionSerializer):
+class MovieSessionListSerializer(serializers.ModelSerializer):
     movie_title = serializers.CharField(source="movie.title", read_only=True)
     cinema_hall_name = serializers.CharField(
         source="cinema_hall.name", read_only=True
     )
     cinema_hall_capacity = serializers.IntegerField(
         source="cinema_hall.capacity", read_only=True
     )
+    tickets_available = serializers.IntegerField(read_only=True)
```

Comment:
The 'tickets_available' field in the MovieSessionListSerializer is not defined in the MovieSession model. Ensure that this field is calculated or derived correctly, or remove it if not needed.